bugbug/_classification.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import numpy as np
+from imblearn.metrics import (
+    geometric_mean_score,
+    make_index_balanced_accuracy,
+    specificity_score,
+)
+from sklearn.metrics.classification import precision_recall_fscore_support
+from sklearn.utils.multiclass import unique_labels
+
+
+def classification_report_imbalanced_values(
+    y_true,
+    y_pred,
+    labels=None,
+    target_names=None,
+    sample_weight=None,
+    digits=2,
+    alpha=0.1,
```

Comment:
If you have made other modifications to the function, you can also remove the parameters we don't use.

If you have kept it basically 1:1, leave it as is

Reply:
I have only removed the formatting supporting code and changed the content of the loop to create the final dict instead of populating the final format.

For reference the initial code is located here: https://github.com/scikit-learn-contrib/imbalanced-learn/blob/master/imblearn/metrics/_classification.py#L750

Reply:
OK, sounds good. Can you add the link (a perma one) to the comment?

Reply:
Sure